configs/storage-proxy/sample.toml

```diff
@@ -1,6 +1,6 @@
 [etcd]
 namespace = "local"                         # env: BACKEND_NAMESPACE
-addr = { host = "127.0.0.1", port = 2379 }  # env: BACKEND_ETCD_ADDR (host:port)
+addr = { host = "127.0.0.1", port = 8121 }  # env: BACKEND_ETCD_ADDR (host:port)
```

Comment:
`2379` is de-facto standard for Etcd server's listening port number. Please just leave it as-is and let's create a separate issue to create a storage-proxy version of `halfstack.toml` config file.

Reply:
sure, i'll revert it..
how about changes in `install-dev.sh`? am i revert that too?

Reply:
Yes please.